packages/gauges/src/getAllGauges.ts

```diff
@@ -0,0 +1,36 @@
+import { CONFIG_PROD, CONFIG_TESTNET } from './constants/config'
+import { publicClient, testnetPublicClient } from './contract'
+import { fetchAllGauges } from './fetchAllGauges'
+import { fetchAllGaugesVoting } from './fetchGaugeVoting'
+import { Gauge } from './types'
+
+export type getAllGaugesOptions = {
+  testnet?: boolean
+}
+
+export const getAllGauges = async (
+  options: getAllGaugesOptions = {
+    testnet: false,
+  },
+): Promise<Gauge[]> => {
+  const { testnet } = options
+  const client = testnet ? testnetPublicClient : publicClient
```

Comment:
I'd recommend to pass in the client as part of the function parameters. It's hard to maintain the nodes in every package